config/software/cpanminus.rb

```diff
@@ -15,21 +15,18 @@
 #
 
 name "cpanminus"
-default_version "1.7004"
+default_version "1.9019"
 
 license "Artistic-2.0"
 license_file "http://dev.perl.org/licenses/artistic.html"
 skip_transitive_dependency_licensing true
 
 dependency "perl"
 
-version "1.7040" do
-  source md5: "4fabebffe22eaaf584b345b082a8a9c1"
-end
+# version_list: url=https://github.com/miyagawa/cpanminus/releases filter=*.tar.gz
 
-version "1.7004" do
-  source md5: "02fe90392f33a12979e188ea110dae67"
-end
+version("1.9019") { source md5: "478b2c21f52145d21405c30fc58759ca" }
```

Comment:
Can we update these checksums to `sha256`?

Reply:
sure, I'll do that.